src/mastra/tools/arxiv.tool.ts

```
@@ -0,0 +1,633 @@
+ */
+async function getPdfParseModule(): Promise<PdfParseFunction> {
+	if (pdfParse !== null) {
+		return pdfParse as PdfParseFunction
+	}
+	if (pdfParseError) {
+		throw pdfParseError
+	}
+
+	try {
+		// Dynamic import to avoid parsing demo content on import
+		const pdfMod = await eval('import("pdf-parse")');
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)